06.bowling_object/lib/game.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  def initialize(marks_text)
+    marks_by_frame = self.class.parse_marks(marks_text)
+    @frames = marks_by_frame.map { |marks| Frame.new(*marks) }
+  end
+
+  def score
+    @frames.each_with_index.sum do |frame, i|
+      final_frame?(i) ? frame.score : score_non_final(i)
```

Comment:
```suggestion
      frame.score(@frames)
```

こんな感じでスコアの計算はFrameクラスの責務にしてしまいたいですね。
ちなみにフレームの番号はnewするタイミングで渡すイメージです。